src/features/embalm/stepContent/hooks/useArchaeologistList.ts

```diff
@@ -21,27 +25,34 @@ export function useArchaeologistList() {
     archaeologists,
     selectedArchaeologists,
     diggingFeesSortDirection,
+    unwrapsSortDirection,
+    failsSortDirection,
+    archsSortDirection,
     diggingFeesFilter,
+    unwrapsFilter,
+    failsFilter,
     archAddressSearch,
+    ShowSelectedArchaeologists,
   } = useSelector(s => s.embalmState);
 
-  const onlineArchaeologists = archaeologists.filter(a => a.isOnline);
+  console.log('ShowSelectedArchaeologists', ShowSelectedArchaeologists);
+  const NUMBER_MOCK_ARCH = 30;
+
+  // Used for generating testing archaeologists
+  const onlineArchaeologists = useMemo(
+    () =>
+      NUMBER_MOCK_ARCH > 0
+        ? mochArchaeologists(NUMBER_MOCK_ARCH)
+        : archaeologists.filter(a => a.isOnline),
+    [NUMBER_MOCK_ARCH]
+  );
```

Comment:
This is a great idea for testing but needs to be removed from the PR.

Reply:
I've comment out the feature if needed in the future

Reply:
That works, but in the future you can just remove it since we can look back in git history if we need it.